### PR TITLE
spaghetti carbonara recipe

### PR DESCRIPTION
**Pull request by gianluccag**
I added a lovely carbonara recipe I learnt in Italy!

### CODE REVIEW

**Review by jmedholm — COMMENTED** [review]
This looks good!

File: gianlucca.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	3.2. grate the cheese fine 
+	3.3. seperate one egg yolk from the white (you can discard one of the egg whites)
+4. Mix in a large bowl the the pancetta, garlic, grated cheese, 1 egg and 1 egg yolk and some olive oil
+5. Add salt, pepper and nutmeg to the mixture to taste
+6. When the pasta is ready, rinse it directly (save some of the pasta water) and add it directly to the mixture in the bowl.
+7. Add a little bit of the pasta water to emulsify.
+8. ENJOY!
+
+Caution: the salt balance is tricky and it can get easily out of hand. The pancetta might have some salt 
+added to it, salt is added to the pasta water and the mixture has salt. BE CAREFUL AND RATHER EXTRA SALT AFTERWARDS
```
Comment (HSR-36) [inline gianlucca.txt]: word missing, And rather ADD extra salt

File: gianlucca.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Ingredients:
 - type 1 or 2 spaghetti
 - 2 eggs
 - pancetta
```
Comment (jmedholm) [inline gianlucca.txt]: Specify the amount

File: gianlucca.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Ingredients:
...
-- 2 garlic cloves
+- 4 garlic cloves
 - olive oil
 - 100 grams parmegiano
 - salt
```
Comment (jmedholm) [inline gianlucca.txt]: - salt to taste

File: gianlucca.txt [hunk 4]
```diff
@@ -4,7 +4,7 @@ Ingredients:
 - type 1 or 2 spaghetti
 - 2 eggs
 - pancetta
-- 2 garlic cloves
+- 4 garlic cloves
 - olive oil
```
Comment (jmedholm) [inline gianlucca.txt]: - a healthy splash of olive oil

**Review by jmedholm — COMMENTED** [review]
Added some comments to this great recipe

File: gianlucca.txt (hunk 4 above)
Comment (gianluccag) [reply gianlucca.txt]: or unhealthy.. to taste

File: gianlucca.txt (hunk 3 above)
Comment (gianluccag) [reply gianlucca.txt]: to state the obvious!

File: gianlucca.txt (hunk 2 above)
Comment (gianluccag) [reply gianlucca.txt]: amount specified

File: gianlucca.txt (hunk 1 above)
Comment (gianluccag) [reply gianlucca.txt]: ADD is ADDED

**Pull request merged**